specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/cdn.json

```diff
@@ -0,0 +1,5553 @@
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "List of CDN operations supported by the CDN resource provider."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of operation list results if there are any."
+        }
+      }
+    },
+    "EdgenodeResult": {
+      "description": "Result of the request to list CDN edgenodes. It contains a list of ip address group and a URL link to get the next set of results.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/EdgeNode"
+          },
+          "description": "Edge node of CDN service."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of edgenode list results if there are any."
+        }
+      }
+    },
+    "EdgeNode": {
+      "description": "Edgenode is a global Point of Presence (POP) location used to deliver CDN content to end users.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/EdgeNodeProperties"
+        }
+      }
+    },
+    "EdgeNodeProperties": {
+      "description": "The JSON object that contains the properties required to create an edgenode.",
+      "required": [
+        "ipAddressGroups"
+      ],
+      "properties": {
+        "ipAddressGroups": {
+          "description": "List of ip address groups.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IpAddressGroup"
+          }
+        }
+      }
+    },
+    "IpAddressGroup": {
+      "description": "CDN Ip address group",
+      "type": "object",
+      "properties": {
+        "deliveryRegion": {
+          "description": "The delivery region of the ip address group",
+          "type": "string"
+        },
+        "ipv4Addresses": {
+          "description": "The list of ip v4 addresses.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/cidrIpAddress"
+          }
+        },
+        "ipv6Addresses": {
+          "description": "The list of ip v6 addresses.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/cidrIpAddress"
+          }
+        }
+      }
+    },
+    "cidrIpAddress": {
+      "description": "CIDR Ip address",
+      "type": "object",
+      "properties": {
+        "baseIpAddress": {
+          "description": "Ip address itself.",
+          "type": "string"
+        },
+        "prefixLength": {
+          "description": "The length of the prefix of the ip address.",
+          "type": "integer"
+        }
+      }
+    },
+    "Resource": {
```

Comment:
ARM requires a new top level properties for all tracked / proxy resources.
https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc027-systemdata-support

You can reference the one in common-type.
https://github.com/Azure/azure-rest-api-specs/blob/a16488bbd0d17551c729e22ea6ac6f7c6588a0a3/specification/common-types/resource-management/v2/types.json#L445